src/flamegraph/svg.rs

```diff
@@ -187,21 +183,20 @@ impl SVGWriter {
             },
         )?;
 
-        match self.subtitle.clone() {
-            Some(subtitle) => self.write_str(
+        if let Some(ref subtitle) = self.subtitle {
+            self.write_str(
                 svg,
                 &mut buf,
                 TextItem {
                     color: "rgb(160, 160, 160)",
                     size: self.font_size,
                     x: (self.image_width / 2) as f64,
                     y: (self.font_size * 4) as f64,
-                    text: subtitle.into(),
+                    text: subtitle.clone().into(),
```

Comment:
See https://github.com/jonhoo/inferno/pull/52#discussion_r259893659. Specifically, this should work:
```rust
text: (&**subtitle).into()
```

Reply:
done